### PR TITLE
dbeaver/pro#1974 update azure UI & detect azure model

### PR DESCRIPTION
**Pull request by Destrolaric**
Closes dbeaver/pro#1974

### CODE REVIEW

File: plugins/org.jkiss.dbeaver.model.ai/src/org/jkiss/dbeaver/model/ai/openai/OpenAICompletionEngine.java [hunk 1]
```diff
@@ -158,6 +158,7 @@ protected String requestCompletion(
         DBSObjectContainer mainObject = getScopeObject(context, executionContext);
 
         final GPTModel model = getModel();
+        GPTCompletionAdapter service = getServiceInstance(executionContext);
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.model.ai/src/org/jkiss/dbeaver/model/ai/openai/OpenAICompletionEngine.java:161]: Why this change?

Comment (Destrolaric) [reply plugins/org.jkiss.dbeaver.model.ai/src/org/jkiss/dbeaver/model/ai/openai/OpenAICompletionEngine.java:161]: Because we requested getMaxTokenSize() before getting info about max tokens from the instance

**Review by LonwoLonwo — APPROVED** [review]

**Review by ShadelessFox — APPROVED** [review]
